cmake/interrogate_setup_dot_py.py

```diff
@@ -92,7 +92,7 @@ def _get_locations(pkgs, package_dir):
     return locations
 
 
-def generate_cmake_file(package_name, version, scripts, package_dir, pkgs, modules):
+def generate_cmake_file(setup_module, package_name, version, scripts, package_dir, pkgs, modules):
```

Comment:
To avoid breaking the API please add the new argument at the end and make it optional (and only add the variable below if a value was passed.

Reply:
Resolved in 00847412